README.md

```diff
@@ -1,2 +1,22 @@
-# react-example
-React Example with the github API
+# React with Redux Example Project
```

Comment:
You were probably planning on doing this already, but just in case not! The Readme is the first thing they'll look at, so I'd put all the info you've got in your PR description in here - testing all markdown/links etc, including all the screen shots and stuff!

Reply:
I'm definitely planning on doing this I was planning on moving the description content to the readme content after I merged it in